src/main/java/com/softserve/teamproject/entity/resource/GroupResource.java

```diff
@@ -0,0 +1,88 @@
+package com.softserve.teamproject.entity.resource;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.softserve.teamproject.entity.deserializer.LocalDateDeserializer;
+import com.softserve.teamproject.entity.deserializer.LocalDateSerializer;
+import com.softserve.teamproject.entity.enums.BudgetOwner;
+import java.time.LocalDate;
+import java.util.Set;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.hateoas.ResourceSupport;
+import org.springframework.stereotype.Component;
+
+/**
+ * Use for represent Group object in http response in HATEOAS style
+ */
+@Component
+public class GroupResource extends ResourceSupport {
+
+  private int groupId;
+
+  private String name;
+
+  @DateTimeFormat(pattern = "dd/MM/yyyy")
+  @JsonDeserialize(using = LocalDateDeserializer.class)
+  @JsonSerialize(using = LocalDateSerializer.class)
+  private LocalDate startDate;
+
+  @DateTimeFormat(pattern = "dd/MM/yyyy")
+  @JsonDeserialize(using = LocalDateDeserializer.class)
+  @JsonSerialize(using = LocalDateSerializer.class)
```

Comment:
> @DateTimeFormat(pattern = "dd/MM/yyyy") 
I'm not sure that this works. You do not need this because of  
> private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
in LocalDateSerializer class.

> @JsonDeserialize(using= LocalDateDeserializer.class)
It never deserializes dates for GroupResource, mb this annotation is not necessary.